modules/extension/xsd/xsd-sld/src/test/java/org/geotools/se/v1_1/SEExampleTest.java

```diff
@@ -221,6 +221,43 @@ public void testParsePointSymbolizer4() throws Exception {
         assertNull(em.getOnlineResource());
     }
 
+    public void testParsePointSymbolizerRelativeURL() throws Exception {
+        /*<PointSymbolizer version="1.1.0" xsi:schemaLocation="http://www.opengis.net/se http://www.opengis.net/se/1.1.0/Symbolizer.xsd" xmlns="http://www.opengis.net/se" xmlns:ogc="http://www.opengis.net/ogc" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" uom="http://www.opengeospatial.org/se/units/pixel">
```

Comment:
This example does not reflect the contents of SLD actually parsed, it's confusing. Either update or remove.